TP/steps/08-ngfor-solution/src/app/people/people.component.ts

```diff
@@ -9,12 +9,12 @@ import { HttpClient } from '@angular/common/http';
 export class PeopleComponent implements OnInit {
   people;
 
-  constructor(private _http: HttpClient) {}
+  constructor(private http: HttpClient) {}
```

Comment:
suggestion(non-blocking): maybe it will be a good practice to make this readonly

Reply:
Done d1837cb33d5424ba6c881cab8cf0b259d3652040

Reply:
this can be done in all exercice